### ISSUE TITLE
Dead link for commitlint-config-conventional

### ISSUE THREAD

**Issue by alexander-mart** [conventional-commits/conventionalcommits.org#446]
Dead link:
[commitlint-config-conventional](https://github.com/marionebl/commitlint/tree/master/%40commitlint/config-conventional)

Page:
https://www.conventionalcommits.org/ru/v1.0.0-beta.2/#спецификация

**Comment by rakovets** [comment]
@alexander-mart Yes, you are right. But I think we shouldn't be looking at the beta. The link is correct for version 1.0.0

**Comment by damianopetrungaro** [comment]
@alexander-mart (or @rakovets ) feel free to end a PR to fix it, a dead link should work also for the older spec version :)
